convert2rhel/unit_tests/systeminfo_test.py

```diff
@@ -131,6 +128,7 @@ def test_modified_rpm_files_diff_without_differences_after_conversion(self):
         data=[['.M.......  g /etc/pki/ca-trust/extracted/java/cacerts'],
               ['.M.......  g /etc/pki/ca-trust/extracted/java/cacerts',
                'S.5....T.  c /etc/yum.conf']]))
+    @skipIf(not is_rpm_based_os(), reason="Current test runs only on rpm based systems.")
```

Comment:
I'm getting the following on CentOS 6 when running the tests with `make tests`:
```
test_modified_rpm_files_diff_with_differences_after_conversion (convert2rhel.unit_tests.systeminfo_test.TestSysteminfo) ... /data/convert2rhel/unit_tests/__init__.py:182: UserWarning: Test <function test_modified_rpm_files_diff_with_differences_after_conversion at 0x234c140> skipped. Reason: Current test runs only on rpm based systems.
  warn("Test %r skipped. Reason: %s" % (test_item, reason))
o
```

Reply:
This will be soved by using @pytest.mark.skipif after #150 will be merged